src/Xamarin.Android.Build.Tasks/Resources/MultiDexLoader.java

```diff
@@ -0,0 +1,254 @@
+package mono.android;
```

Comment:
Given that, at least in *concept*, much of the contents of this file came from `StubApplication.java`, we should *probably* preserve the same copyright header that `StubApplication.java` had.

"Even better," perhaps we should `git mv StubApplication.java MultiDexLoader.java`, and *expunge* all other references to/uses of `StubApplication.java`?

Reply:
Header has been moved.